sdk/storage/storage-blob/test/node/blobclient.spec.ts

```diff
@@ -58,6 +58,14 @@ describe("BlobClient Node.js only", () => {
     await recorder.stop();
   });
 
+  before(async function() {
+    recorder = record(this, recorderEnvSetup);
+    if (!existsSync(tempFolderPath)) {
+      mkdirSync(tempFolderPath);
+    }
+    await recorder.stop();
+  });
```

Comment:
No HTTP request is being made here..
```suggestion
  before(async function() {
    if (!existsSync(tempFolderPath)) {
      mkdirSync(tempFolderPath);
    }
  });
```
should be enough. (And the corresponding before_all empty recording can be deleted)